src/lib/ui/noise-sensor/NoiseThresholdsList.tsx

```diff
@@ -0,0 +1,140 @@
+import type { NoiseSensorDevice, NoiseThresholds } from 'seamapi'
+import { ZonedTime } from 'zoned-time'
+
+import { formatTime, formatTimeZone } from 'lib/dates.js'
+import { ArrowRightIcon } from 'lib/icons/ArrowRight.js'
+import { useNoiseThresholds } from 'lib/seam/noise-sensors/use-noise-thresholds.js'
+import { DetailRow } from 'lib/ui/layout/DetailRow.js'
+import { DetailSection } from 'lib/ui/layout/DetailSection.js'
+import { DetailSectionGroup } from 'lib/ui/layout/DetailSectionGroup.js'
+
+interface NoiseThresholdsListProps {
+  device: NoiseSensorDevice
+}
+
+export function NoiseThresholdsList({
+  device,
+}: NoiseThresholdsListProps): JSX.Element {
+  const { noiseThresholds, isInitialLoading } = useNoiseThresholds({
+    device_id: device.device_id,
+  })
+
+  return (
+    <>
+      <DetailSectionGroup>
+        <div className='seam-detail-section-wrap'>
+          <DetailSection
+            label={t.noiseThresholds}
+            tooltipContent={
+              device.device_type === 'minut_sensor' ? (
```

Comment:
Why does `minut` have special behavior here?

Reply:
Not sure—it is specified in the design: https://www.figma.com/file/WTdzEh4CudGdQFOtuVR3Eh/Thermostat%2FNoise-components?type=design&node-id=3996-10671&mode=design&t=GfLxqO8DBwAaTIaZ-4